src/components/allNews/AllNews.jsx

```diff
@@ -0,0 +1,41 @@
+'use client'
+import React, { useState } from 'react'
+import NewsBox from '@/components/newsBox/NewsBox'
+import { formatDate } from '@/utils/formatters'
+import styles from './allNews.module.css'
+
+const AllNews = ({ posts }) => {
+  const [loadedPostsAmount, setloadedPostsAmount] = useState(6)
+  const loadedPosts = posts.slice(0, loadedPostsAmount)
+  const handleShowMore = () => {
+    setloadedPostsAmount((prev) => (prev += 6))
+  }
```

Comment:
this will be done in a different way, but for now let's keeps it as it is

Reply:
okay